byteps/tensorflow/ops.py

```diff
@@ -80,7 +80,12 @@ def _push_pull(tensor, scope='', name=None):
     if name is None and not _executing_eagerly():
         name = 'BytePSPushPull_%s' % _normalize_name(tensor.name)
     if scope == '' and not _executing_eagerly():
-        scope = tf.compat.v1.get_default_graph().get_name_scope()
+        try:
+            scope = tf.get_default_graph().get_name_scope()
+        except:
```

Comment:
Put a specific exception case after `except`?

Reply:
Perhaps if-else will be better

Reply:
agreed